### PR TITLE
add a blog section on `code.groundlight.ai` and a blog post on framegrab

### PR DESCRIPTION
**Pull request by blaise-muhirwa**
This adds a blog in our docusaurus docs. The blog that I added currently looks like this: 

Screenshot 2023-12-06 at 3 51 15 PM
Screenshot 2023-12-06 at 3 51 28 PM
Screenshot 2023-12-06 at 3 51 34 PM

### CODE REVIEW

File: docs/blog/2023-12-06-framegrab.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    if motion_detector.motion_detected(frame):
+        print("Motion detected!")
+
+```
+
+
+## Conclusion 
+
+
+This release of FrameGrab comes with several awesome features that are very easy to use. We now support
```
Comment (tyler-romero) [inline docs/blog/2023-12-06-framegrab.md:171]: Trying to reduce the repetition of the word 'awesome'
```suggestion
This release of FrameGrab adds some great, easy to use features. We now support 
```

**Review by tyler-romero — APPROVED** [review]
LGTM with some minor nits!

What makes it hard to change the url?

File: docs/blog/2023-12-06-framegrab.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+---
+title: Announcing Groundlight's FrameGrab Library
+description: Today, we are happy to announce FrameGrab 0.4.3.
```
[the remark this answers is not in the document]
Comment (tyler-romero) [reply docs/blog/2023-12-06-framegrab.md:3]: What are the v0.4.3 specific features?

File: docs/blog/2023-12-06-framegrab.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        digital: 1.5
+"""
+
+grabber = FrameGrabber.create_grabber_yaml(config)
+frame = grabber.grab()
+
+# Do real work with the frame 
+
+# Finally release the grabber object 
+grabber.release()
```
[the remark this answers is not in the document]
Comment (blaise-muhirwa) [reply docs/blog/2023-12-06-framegrab.md:63]: Good suggestion!

File: docs/blog/2023-12-06-framegrab.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+`input_type` include 
+
+* generic_usb
+* rtsp
+* realsense
+* basler 
+
+Here is an example of how to use the generic USB configuration 
+
+```python
```
[the remark this answers is not in the document]
Comment (blaise-muhirwa) [reply docs/blog/2023-12-06-framegrab.md:41]: I went with the `python notest` option.

**Pull request merged**